### PR TITLE
Update pnpm version in CI

### PR DESCRIPTION
**Pull request by DukeManh**
## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: .github/workflows/e2e-tests-ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
 
       - uses: pnpm/action-setup@v2.1.0
         with:
-          version: 6
```
Comment (humphd) [inline .github/workflows/e2e-tests-ci.yml:26]: I think we should let the [`package.json` determine this](https://github.com/pnpm/action-setup#version):

> Optional when there is a [packageManager field in the package.json](https://nodejs.org/api/corepack.html).

**Review by humphd — CHANGES_REQUESTED** [review]

**Review by Kevan-Y — APPROVED** [review]

File: .github/workflows/e2e-tests-ci.yml (hunk 1 above)
Comment (DukeManh) [reply .github/workflows/e2e-tests-ci.yml:26]: This option can go then.

**Comment by Kevan-Y** [comment]
@DukeManh , do we want to update `pnpm/action-setup` to `v2.1.0`. The older one still use `v2.0.1`?

**Comment by DukeManh** [comment]
@Kevan-Y Renovate is updating it  https://github.com/Seneca-CDOT/telescope/pull/3425#issuecomment-1092058965

File: .github/workflows/e2e-tests-ci.yml (hunk 1 above)
Comment (Kevan-Y) [reply .github/workflows/e2e-tests-ci.yml:26]: Only  ` pnpm/action-setup@v2.2.0` above support that https://github.com/marketplace/actions/setup-pnpm?version=v2.2.0

**Review by humphd — APPROVED** [review]
Awesome, now we don't need to keep updating all the time.

**Review by Kevan-Y — APPROVED** [review]

**Pull request merged**